src/client.ts

```diff
@@ -419,6 +423,10 @@ export class Client {
         let headers = context.init.headers as Record<string, string>;
         headers['X-Adzerk-Sdk-Version'] = versionString;
```

Comment:
On the other hand I'm not sure whether we support `X-Kevel-Sdk-Version`. I don't think we do, actually.

Just checked. Nope. We should at some point.